getting-started-with-directus-and-flask/index.md

```diff
@@ -0,0 +1,396 @@
+import directus // [!code ++]
+
+app = Flask(__name__)
+
+
+@app.get("/")
+def home():
+    return "Hello world" // [!code --]
+    global_data = directus.get_global_data() // [!code ++]
+// [!code ++]
+    return render_template( // [!code ++]
+        "home.html", title=global_data["title"], description=global_data["description"] // [!code ++]
+    ) // [!code ++]
+```
+
+Then go to [http://localhost:3000](http://localhost:3000) and you will see a page like this:
+
+![Home page displaying configured global data](home-page.png)
+
+
+## Creating Pages With Directus
+
+Note that the base template has some links on its navbar pointing to pages like **About** that typically have just some text content and some images, but also need to be updated frequently and by people who do not have access to the code base.
+
+Using Directus you can make these pages dynamically available on your site and editable by non-technical users.
+
+To accomplish this, create a new collection called `pages` - make the Primary ID Field a "Manually Entered String" called `slug`, which will correlate with the URL for the page. For example, `about` will later correlate to the page `localhost:3000/about`.
+
+Create a text input field called `title` and a WYSIWYG input field called `content`. In Access Control, give the Public role read access to the new collection. Create 3 items in the new collection - [here's some sample data](https://github.com/directus-community/getting-started-demo-data).
+
+### Rendering Dynamic Pages
+
+To get data of pages registered on the `pages` collection you will need to add the following code at the end of the *directus.py* file:
+
+```py
+def get_page_by_slug(slug):
+    response = requests.get(f"{DIRECTUS_BASE_URL}/items/pages/{slug}")
+    return response.json().get("data")
+```
+
+Create the *templates/dynamic-page.html* file with the following content:
+
+```jinja
+{% extends "base.html" %}
+
+{% block content %}
+	<h1>{% block title %}{{ title }}{% endblock %}</h1>
+	<article>{{ content | safe }}</article>
+{% endblock %}
+```
+
+Then, on the *app.py* file import `render_template_string` from `Flask` and define a new app route with the following code at the end of the file:
+
+```py
+@app.get("/<slug>")
+def dynamic_page(slug):
+    page = directus.get_page_by_slug(slug)
+
+    if not page:
+        return render_template_string(
+            "{% extends 'base.html' %}{% block content %}This page does not exists{% endblock %}"
+        )
+
+    return render_template(
+        "dynamic-page.html", title=page["title"], content=page["content"]
+    )
+```
+
+This route fetches page data using the `directus.get_page_by_slug` method and then renders a simple not found page (defined as an inline template string) if the page does not exist, and if it exists it renders the `dynamic-page.html` template with page data on it.
+
+Now navigate to [http://localhost:3000](http://localhost:3000) and see the result
```

Comment:
Same comment here - can you change these to backtick non-links throughout please?